BUILD_LINUX.md

```diff
@@ -118,33 +152,30 @@ Prepare makefiles:
 cmake ..
 ```
 
-- If cmake fails with a vcpkg error - delete /tmp/hifi/vcpkg.  
+If cmake fails with a vcpkg error, then delete `/tmp/vircadia/vcpkg`.  
```

Comment:
I have no info of the /tmp/ directly ever being used for vcpkg, but if you run into vcpkg issues chances are you are hitting a cache that you shouldn't hit (like currently when you try to build with GLES and without GLES without changing the source in between). That cache is in ~/vircadia-files/vcpkg/
```suggestion
If cmake fails with a vcpkg error, then delete `~/vircadia-files/vcpkg/`.  
```